src/config.c

```diff
@@ -1060,52 +797,14 @@ void configGetCommand(client *c) {
     }
 
     /* String values */
-    config_get_string_field("dbfilename",server.rdb_filename);
     config_get_string_field("logfile",server.logfile);
-#ifdef USE_OPENSSL
-    config_get_string_field("tls-cert-file",server.tls_ctx_config.cert_file);
-    config_get_string_field("tls-key-file",server.tls_ctx_config.key_file);
-    config_get_string_field("tls-dh-params-file",server.tls_ctx_config.dh_params_file);
-    config_get_string_field("tls-ca-cert-file",server.tls_ctx_config.ca_cert_file);
-    config_get_string_field("tls-ca-cert-dir",server.tls_ctx_config.ca_cert_dir);
-    config_get_string_field("tls-protocols",server.tls_ctx_config.protocols);
-    config_get_string_field("tls-ciphers",server.tls_ctx_config.ciphers);
-    config_get_string_field("tls-ciphersuites",server.tls_ctx_config.ciphersuites);
-#endif
 
     /* Numerical values */
-    config_get_numerical_field("maxmemory",server.maxmemory);
     config_get_numerical_field("client-query-buffer-limit",server.client_max_querybuf_len);
-    config_get_numerical_field("auto-aof-rewrite-min-size",
-            server.aof_rewrite_min_size);
-    config_get_numerical_field("stream-node-max-entries",
-            server.stream_node_max_entries);
-    config_get_numerical_field("tls-port",server.tls_port);
-    config_get_numerical_field("repl-backlog-size",server.repl_backlog_size);
-    config_get_numerical_field("repl-backlog-ttl",server.repl_backlog_time_limit);
-    config_get_numerical_field("maxclients",server.maxclients);
     config_get_numerical_field("watchdog-period",server.watchdog_period);
-    config_get_numerical_field("min-slaves-to-write",server.repl_min_slaves_to_write);
-    config_get_numerical_field("min-replicas-to-write",server.repl_min_slaves_to_write);
-    config_get_numerical_field("min-slaves-max-lag",server.repl_min_slaves_max_lag);
-    config_get_numerical_field("min-replicas-max-lag",server.repl_min_slaves_max_lag);
-    config_get_numerical_field("hz",server.config_hz);
-
-    /* Bool (yes/no) values */
-    config_get_bool_field("activedefrag", server.active_defrag_enabled);
```

Comment:
Removing these means config_get_bool_field is now dead code.

Reply:
considering we still have some custom configs left (for other types, not bool), i think i'll leave that dead code for now (part of a macro system that's still in used), maybe we'll need it later.